R/functions_output.R

```diff
@@ -1,124 +1,33 @@
 ######################### ###
 #
-# Print, summary and xtable functions
+# Print and summary functions
```

Comment:
Alternatively, we could just tidy print straight away?

Reply:
I don't follow this one

Reply:
That is, make the default printing the tidy printing.

Reply:
Right now `tidy()` uses `summary()` to get the `coefMat` matrix.

I prefer to keep it like it's now.